go.project/src/org/tvl/goworks/project/GoSourceNodeFactory.java

```diff
@@ -196,6 +202,8 @@ public PackageViewFilterNode(SourceGroupKey sourceGroupKey, Project project) {
                 }
             }
 
+            actions.add(FileSensitiveActions.fileCommandAction(GoActionProvider.COMMAND_DOWNLOAD_DEPENDENCIES, Bundle.LBL_DownloadDependencies_Action(), null));
```

Comment:
:bulb: This places the command on the **src** node, which it should probably be placed on the project node.

:bulb: This should be located above **Properties**, not below it.